src/net.cpp

```diff
@@ -367,8 +367,10 @@ static CAddress GetBindAddress(SOCKET sock)
     return addr_bind;
 }
 
-CNode* CConnman::ConnectNode(CAddress addrConnect, const char *pszDest, bool fCountFailure, bool manual_connection, bool block_relay_only)
+CNode* CConnman::ConnectNode(CAddress addrConnect, const char *pszDest, bool fCountFailure, ConnectionType conn_type)
```

Comment:
`conn_type` is correct. See the developer guide: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c

Reply:
Thanks for the pointer, I was wondering about the inconsistency between variables (`addrConnect` instead of `addr_connect`), but this probably falls under `These are preferred in new code, but are not required when doing so would need changes to significant pieces of existing code.`